markdown/setup/tableau-desktop-driver-setup.md

```diff
@@ -0,0 +1,14 @@
+### Tableau Desktop
+[Link to product webpage](https://www.tableau.com/products/desktop).
+
+#### Using Tableau with Generic JDBC driver (without using Connector).
+1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver fully shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`) and copy it to one of these
+   directories according to your operating system:
+   - **_Windows_**: `C:\Program Files\Tableau\Drivers`
+   ![Example](../images/tableau/tableau-driver-location.png)
+    - **_macOS_**: `~/Library/Tableau/Drivers`
+
+2. Open Tableau > Connect > Other Databases (JDBC)
```

Comment:
Need to explain how Access Key, Secret Key, and Region are configured since it is not possible to specify them in this dialog window.

Are we suggesting adding them in the URL?

Reply:
Access Key etc can be configured by specifying them in the dialog window. I added instructions for adding connection properties (see step # 3).